### DISCUSSION TITLE
How to access account

### DISCUSSION THREAD

**Discussion by SONA-LK** [community/community#124061] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I have accidently unlink email and now i can't access to account.please help me to recover it.

**Comment by talhaahussain** [comment]
If you know the username of the account, you should be able to sign in using the username instead of the email:
<br>
![image](https://github.com/community/community/assets/73853725/eefd4f5f-d45b-4e03-8fe9-5a13002cdd23)

Once you are signed into the account, you can relink the email by clicking on your icon in the top right corner, going to "Settings", and then clicking "Emails" on the left panel. You'll then have an option to "Add email address", which you can use to relink your email address.

**Comment by Atharvkote** [comment]
Sign in using User name Mate!!! and then again add emails address!! Simple as that !
And Hii If Your are Free Please mark this anser as answered ❤️
